tests/it.rs

```diff
@@ -234,3 +244,24 @@ fn test_capabilities() {
         .contains("cap_net_admin=ep"));
     assert!(status.success());
 }
+
+#[test]
+fn test_no_auto_etc_os_release() {
```

Comment:
Consider adding a test for the scenario where /etc/os-release is added as an os package dependency.